src/suricata.c

```diff
@@ -2308,11 +2308,11 @@ void PreRunInit(const int runmode)
  * but after we dropped privs */
 void PreRunPostPrivsDropInit(const int runmode)
 {
+    RunModeInitializeOutputs();
     if (runmode == RUNMODE_UNIX_SOCKET)
         return;
 
     StatsSetupPostConfigPreOutput();
```

Comment:
Should this call precede the call to `RunModeInitializeOutputs`?

Reply:
Changed